src/doc/conf.py

```diff
@@ -55,17 +55,17 @@
 
 # General information about the project.
 project = u'VisIt User Manual'
-copyright = u'2008-2017, LLNL, UCRL-SM-220449'
+copyright = u'2008-2019, LLNL, UCRL-SM-220449'
 author = u'LLNL'
 
 # The version info for the project you're documenting, acts as replacement for
 # |version| and |release|, also used in various other places throughout the
 # built documents.
 #
 # The short X.Y version.
-version = u'2.11'
+version = u'@VISIT_VERSION_MAJOR@.@VISIT_VERSION_MINOR@'
```

Comment:
Just noticed lc_sphnx_build_command.sh, and I think my changes here might cause problems for that script. Perhaps we need a conf.py.in for use within our builds (to use with configure_file in doc\CMakeLists.txt.

Reply:
I reverted my '@' changes and theme changes to conf.py.  Modified the CMakeLists to pass the theme and version via arguments instead and did away with the use of configure-file to modify conf.py

Reply:
I tested this on Windows and linux, and 'make gui_manual' produces good themed output from a development build of visit.